Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -440,7 +440,7 @@ def bottle_formula(f, args:)
 
         # Ignore matches to source code, which is not required at run time.
         # These matches may be caused by debugging symbols.
-        ignores = [%r{/include/|\.(c|cc|cpp|h|hpp)$}]
+        ignores = [%r{/include/|\.(c|cc|cpp|h|hpp)$|#{cellar}/gcc}]
```

Comment:
```suggestion
        ignores = [%r{/include/|\.(c|cc|cpp|h|hpp)$}]

        # GCC installation can be moved so long as the whole directory tree is moved together (which we are doing here): 
        # https://gcc-help.gcc.gnu.narkive.com/GnwuCA7l/moving-gcc-from-the-installation-path-is-it-allowed.
        if Formula["gcc"].versioned_formulae.map(&:name).include?(name)
          ignores << %r{#{Regexp.escape(cellar)}/gcc/}
        end

```

Reply:
Thanks, this is super helpful!  One quick question, will `Formula["gcc"].versioned_formulae.map` also include the current version of GCC (GCC 11)?  Because we'd want to include that as well.

Reply:
yes

Reply:
I made some progress in figuring out why `gcc@5` was missing from the list of versioned formulae produced by `Formula.versioned_formulae`.  It is **not** specific to `gcc@5`.  When I tried to use `brew bottle` with `gcc@9`, for example, it skipped `gcc@9` but did load the other versions, including `gcc@5`.  I am puzzled as to how the version of `gcc` I'm bottling with `brew bottle` could have any influence on the output of `versioned_formulae`, given that we are calling the method for `Formula["gcc"]`, and not the specific version of `gcc` being bottled.  

I am wondering if this is somehow related to `next if versioned_path == path` in 
```
  def versioned_formulae
    Pathname.glob(path.to_s.gsub(/(@[\d.]+)?\.rb$/, "@*.rb")).map do |versioned_path|
      next if versioned_path == path

      Formula[versioned_path.basename(".rb").to_s]
    rescue FormulaUnavailableError
      nil
    end.compact.sort_by(&:version).reverse
  end
```
but when I comment that line out, the result is the same.

Reply:
@danielnachun Thanks for digging in. Think it's fine to just hack around this for now.